pegasus/sites.v3/code.org/views/professional_learning_apply_banner.haml

```diff
@@ -16,8 +19,10 @@
         .arrowcontainer.desktop-feature{style: "float:left; width: 7%"}
           .arrow{style: "width: 0; height: 0; border-style: solid; border-width: 50px 0 50px 25px; border-color: transparent transparent transparent #ebe8f1"}
         .textcontainer{style: "float:left; width: 93%"}
-          .text{style: "color: white;  font-size: 18px; padding: 20px 10px 20px 10px; text-align: center; line-height: 1.5"}
-            - if params[:nominated]
+          .text{style: "color: white;  font-size: #{font_size}; padding: #{padding}; text-align: center; line-height: 1.5"}
+            - if use_sign_up_text
+              Sign up for Professional Learning! Applications now open for Middle and High School teachers.
```

Comment:
Something I should have caught in the last PR: There's no verb in the second sentence.  @agealy can we change this to "Applications _are_ now open..."?

Reply:
Sure, I'm ambivalent. The current verbless phrasing was what came through the marketing and programs approved request

Reply:
would we want to update the homepage text as well? (it's the same "Applications now open...")

Reply:
:+1: I vote yes.

Reply:
updated text